### DISCUSSION TITLE
xmake的exe文件运行需要管理员权限,希望免除管理员权限

### DISCUSSION THREAD

**Discussion by sosoeo** [xmake-io/xmake#2829] [Q&A]
### Xmake Version

2.7

### Operating System Version and Architecture

windows10

### Describe Bug

windows平台的安装文件(exe),需要管理员权限才能运行,在企业环境下非常不友好,导致无法运行.能否设计成像vscode那样免管理员权限运行?或免安装运行也可以??

### Expected Behavior

免管理员权限运行,像vscode那样

### Project Configuration

default

### Additional Information and Error Logs

default

**Comment by waruqi** [comment]
我这不会，也从来没其他用户报过这个问题，都能直接跑。。暂时无法测试验证，你可以自己拉下代码 改进下提个 pr 过来

**Comment by sosoeo** [comment]
> 我这不会，也从来没其他用户报过这个问题，都能直接跑。。暂时无法测试验证，你可以自己拉下代码 改进下提个 pr 过来

在win10中,需要用户不是admin的时候,只是admin分配的普通账户的时候,比如我现在的工作电脑,运行exe时,会要求输入管理员密码,这个只有IT介入才行.vscode 安装的时候就不需要管理员权限,chrome浏览器安装的时候也不需要管理员权限
> 我这有个管理员权限要求的弹窗,这里不能贴图.(xmake 2.7.1 setup this installer requires admin privileges,aborting!)

**Comment by waruqi** [comment]
没遇到过 这边 win10 正常运行，也是普通用户

**Comment by xq114** [comment]
理论上你安装的时候装在用户目录下就没问题，装在全局目录program files肯定不行
